src/Selda/PG.purs

```diff
@@ -97,11 +127,8 @@ query q = do
     (Tuple res st') = runQuery $ unwrap q
     st = st' { cols = getCols res }
     q_str = showState st
-  -- liftEffect $ log q_str
```

Comment:
Maybe instead of `liftEffect $ log q_str` we can introduce `logger: String -> Effect Unit` to execution environment of our Selda monad and run it in `pgExecute` and `pgQuery` above. This kind of API will easily integrate with purescript-logging too ;-) What do you think @Kamirus?